CI sometimes is broken

In the CI logs, it is showing that rust-criu version 0.2.0 is used? even though all cargo.toml and lockfile has version 0.5.7 specfified in it?
```
   Compiling procfs v0.14.0
   Compiling rust-criu v0.2.0
error[E0583]: file not found for module `rpc`
 --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/rust-criu-0.2.0/src/proto/mod.rs:1:1
```
The crates.io only has v 0.2 published : https://crates.io/crates/rust-criu , so there might be some mismatch somewhere, although I can't see why it is failing now, and didn't fail before

--- a/.github/workflows/integration_tests_validation.yaml
+++ b/.github/workflows/integration_tests_validation.yaml
@@ -26,7 +26,7 @@ jobs:
     timeout-minutes: 15
     strategy:
       matrix:
-        rust: [1.61.0, 1.62.1]
+        rust: [1.62.1, 1.63.0]
     steps:
       - uses: actions/checkout@v2
       - uses: actions-rs/toolchain@v1

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -31,7 +31,7 @@ jobs:
     timeout-minutes: 15
     strategy:
       matrix:
-        rust: [1.61.0, 1.62.1]
+        rust: [1.62.1, 1.63.0]
         dirs: ${{ fromJSON(needs.changes.outputs.dirs) }}
     steps:
       - uses: actions/checkout@v2
@@ -55,7 +55,7 @@ jobs:
     timeout-minutes: 15
     strategy:
       matrix:
-        rust: [1.61.0, 1.62.1]
+        rust: [1.62.1, 1.63.0]
     steps:
       - uses: actions/checkout@v2
       - uses: actions-rs/toolchain@v1
@@ -105,7 +105,7 @@ jobs:
     timeout-minutes: 15
     strategy:
       matrix:
-        rust: [1.61.0, 1.62.1]
+        rust: [1.62.1, 1.63.0]
     steps:
       - uses: actions/checkout@v2
         with:

--- a/crates/libcontainer/src/config.rs
+++ b/crates/libcontainer/src/config.rs
@@ -14,7 +14,7 @@ const YOUKI_CONFIG_NAME: &str = "youki_config.json";
 
 /// A configuration for passing information obtained during container creation to other commands.
 /// Keeping the information to a minimum improves performance.
-#[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Eq)]
 #[non_exhaustive]
 pub struct YoukiConfig {
     pub hooks: Option<Hooks>,

--- a/crates/youki/Cargo.toml
+++ b/crates/youki/Cargo.toml
@@ -38,4 +38,4 @@ serial_test = "0.9.0"
 
 [build-dependencies]
 anyhow = "1.0.60"
-vergen = "7.3.2"
+vergen = "7.2.1"

--- a/tests/rust-integration-tests/integration_test/src/tests/lifecycle/container_create.rs
+++ b/tests/rust-integration-tests/integration_test/src/tests/lifecycle/container_create.rs
@@ -8,13 +8,13 @@ pub struct ContainerCreate {
     container_id: String,
 }
 
-impl<'a> Default for ContainerCreate {
+impl Default for ContainerCreate {
     fn default() -> Self {
         Self::new()
     }
 }
 
-impl<'a> ContainerCreate {
+impl ContainerCreate {
     pub fn new() -> Self {
         let id = generate_uuid();
         let temp_dir = prepare_bundle(&id).unwrap();

